website/docs/r/s3tables_table.html.markdown

```diff
@@ -105,6 +154,35 @@ The `iceberg_snapshot_management.settings` object supports the following argumen
 * `min_snapshots_to_keep` - (Required) Minimum number of snapshots to keep.
   Must be at least `1`.
 
+### `metadata`
+
+The `metadata` configuration block supports the following argument:
+
+* `iceberg` - (Optional) Contains details about the metadata for an Iceberg table. This block defines the schema structure for the Apache Iceberg table format.
+  [See `metadata.iceberg` below](#metadataiceberg).
+
+### `metadata.iceberg`
```

Comment:
Can we update these field headings to remove the parent address? e.g. just `iceberg` here. Associated links will also need adjusted.